src/main/java/calculator/ArithmeticExpressionStack.java

```diff
@@ -0,0 +1,46 @@
+package calculator;
+
+
+interface Stack{
+    boolean isEmpty();
+    String pop();
+    void clear();
+}
+
+public class ArithmeticExpressionStack implements Stack {
+    private int stackSize;
+    private int flag;
+    private String stack[];
+
+    public ArithmeticExpressionStack(String[] equation, int stackSize){
+        clear();
+        flag = 0;
+        this.stackSize = stackSize;
+        stack = equation;
+    }
+
+    public String pop(){
+        if(isEmpty()){
+            return "스택이 존재하지 않습니다.";
+        }
+        else{
+            return stack[flag++];
+        }
+    }
+
+    public boolean isEmpty(){
+        return (flag == this.stackSize);
+    }
+
+    public void clear(){
+        if(!isEmpty()){
+            flag = -1;
+            this.stackSize = stackSize;
+        }
+    }
+
+    public int getStackSize(){
+        return this.stackSize;
```

Comment:
```suggestion
        return stackSize;
```

```Java
public ArithmeticExpressionStack(String[] equation, int stackSize){
        this.stackSize = stackSize;
    }
```
이 경우와 같이 클래스의 속성과 메소드의 매개변수의 이름이 같은경우가 아니므로 굳이 `this.` 을 써주지 않아도 될 것 같아요~